templates/definition/meter/sonnenbatterie_eco56.yaml

```diff
@@ -0,0 +1,70 @@
+template: sonnenbatterie-eco56
+products:
+  - brand: Sonnen
+    description:
+      generic: comfort, eco 5, eco 6, oem 6.5
+params:
+  - name: usage
+    choice: ["grid", "pv", "battery"]
+  - name: host
+  - name: port
+    default: 7979
+  - name: capacity
+    advanced: true
+render: |
+  type: custom
+  {{- if eq .usage "grid" }}
+  power:
+    source: calc
```

Comment:
You do not need this `calc` operation here as both values are taken from same JSON document.
You can just do `jq: .M39 - .M38`
jq: